src/chain/factory/block/index.ts

```diff
@@ -30,20 +30,20 @@ export async function assembleBlock(
     stateRoot: parentBlock.stateRoot,
     signature: parentBlock.signature,
     slot: parentBlock.slot,
-    previousBlockRoot: parentBlock.previousBlockRoot,
-    blockBodyRoot: hashTreeRoot(parentBlock.body, BeaconBlockBody),
+    parentRoot: parentBlock.parentRoot,
+    bodyRoot: hashTreeRoot(parentBlock.body, BeaconBlockBody),
   };
   const block: BeaconBlock = {
     slot,
-    previousBlockRoot: signingRoot(parentHeader, BeaconBlockHeader),
+    parentRoot: signingRoot(parentHeader, BeaconBlockHeader),
     signature: undefined,
     stateRoot: undefined,
     body: await assembleBody(opPool, currentState, randao),
   };
 
   //This will effectively copy state so we avoid modifying existing state
   const nextState = {...currentState};
-  processBlock(nextState, block, false);
+  processBlock(nextState, block);
```

Comment:
leave verify false flag here. It will fail otherwise because stateRoot and signatures are empty

Reply:
changed submodule url and updated processBlock with verify flag